libminifi/src/c2/ControllerSocketProtocol.cpp

```diff
@@ -51,18 +49,13 @@ void ControllerSocketProtocol::initialize(core::controller::ControllerServicePro
   }
   if (nullptr == secure_context) {
     std::string secureStr;
-    bool is_secure = false;
-    if (configuration->get(Configure::nifi_remote_input_secure, secureStr) && org::apache::nifi::minifi::utils::StringUtils::StringToBool(secureStr, is_secure)) {
+    if (configuration->get(Configure::nifi_remote_input_secure, secureStr) && org::apache::nifi::minifi::utils::StringUtils::toBool(secureStr).value_or(false)) {
       secure_context = std::make_shared<minifi::controllers::SSLContextService>("ControllerSocketProtocolSSL", configuration);
       secure_context->onEnable();
     }
   }
 
-  std::string value;
-
-  if (configuration_->get("controller.socket.local.any.interface", limitStr)) {
-    utils::StringUtils::StringToBool(limitStr, anyInterface);
-  }
+  const bool anyInterface =(configuration_->get("controller.socket.local.any.interface", limitStr) && utils::StringUtils::toBool(limitStr).value_or(false));
```

Comment:
formatting: remove the outermost parentheses and ensure that there is 1 space on both sides of the operator `=`.

https://google.github.io/styleguide/cppguide.html#Horizontal_Whitespace

Reply:
That also decreases the chance of depression as it eliminates a sad smile.